pymodelextractor/tests/learners_tests/test_generic_lstar_learner.py

```diff
@@ -0,0 +1,84 @@
+import time
+import unittest
+
+from pymodelextractor.teachers.generic_teacher import \
+    GenericTeacher
+from pymodelextractor.factories.lstar_factory import LStarFactory
+from pythautomata.automata_definitions.tomitas_grammars import TomitasGrammars
+from pythautomata.model_comparators.hopcroft_karp_comparison_strategy import \
+    HopcroftKarpComparisonStrategy as ComparisonStrategy
+from pythautomata.model_comparators.moore_machine_comparison_strategy import MooreMachineComparisonStrategy
+from pythautomata.model_comparators.dfa_comparison_strategy import DFAComparisonStrategy
+from pythautomata.utilities.automata_converter import AutomataConverter
+from pythautomata.utilities.simple_dfa_generator import generate_dfa
+from pymodelextractor.teachers.pac_comparison_strategy import PACComparisonStrategy
+from pymodelextractor.utils.time_bound_utilities import is_unix_system
+from itertools import chain
+
+
+class TestGenericLStarLearner(unittest.TestCase):
+    def test_generic_learner(self):
```

Comment:
this hole test suite is way too wide, try keeping it under a soft limit of 100 chars per line